app/src/main/java/movierating/services/ReviewServiceImpl1.java

```diff
@@ -39,6 +38,7 @@ public void addReview(String userName, String movieName, int rating) throws Mult
             throw new MovieNotReleasedException("Movie is not yet Released!!");
```

Comment:
Static message part must be inside the exception class. Only the variable part must be passed as argument.

Reply:
okay